inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1311,3 +1310,15 @@ MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>
 
     return newNode;
 }
+
+bool MKLDNNNode::canBePerformedAsScaleShift() const {
+    bool inputsIsConst = true;
+    for (size_t i = 1; i < getParentEdges().size(); i++) {
+        if (!getParentEdgeAt(i)->getParent()->isConstant() || getParentEdgeAt(i)->getParent()->getType() != Input) {
+            inputsIsConst = false;
```

Comment:
just `return false` instead of using special variable.

Reply:
fixed